Bio/SeqIO/GckIO.py

```diff
@@ -14,6 +14,7 @@
 from struct import unpack
 
 from Bio import Alphabet
+from Bio._utils import read_header
```

Comment:
Either:

```python
from Bio._utils import read_header as _read_header
```

Or, rename the function and just:

```python
from Bio._utils import _read_header
```

Otherwise you are adding ``read_header`` as a public part of the ``GckIO`` API.

(Same applies to ``XdnaIO.py``)